addons/website_slides/static/tests/tours/slide_course_publisher_standard.js

```diff
@@ -63,7 +63,8 @@ tour.register('course_publisher_standard', {
     content: 'eLearning: is the Corgi set ?',
     trigger: 'img.o_wslides_course_pict',
     run: function () {
-        if ($('img.o_wslides_course_pict').attr('src').endsWith('GoldWinnerPembrookeWelshCorgi.jpg')) {
+        let regex=/GoldWinnerPembrookeWelshCorgi.jpg/;
+        if ($('img.o_wslides_course_pict').attr('src').match(regex)) {
```

Comment:
I think this is more straightforward, we don't need the array created with `match`
```suggestion
        if (regex.test($('img.o_wslides_course_pict').attr('src'))) {
```

Reply:
Makes total sense, I changed everything accordingly :)